v5/ooapi.v5.core.UnitTests/Repositories/OrganizationsRepositoryTests.cs

```diff
@@ -0,0 +1,176 @@
+using AutoFixture;
+using Microsoft.EntityFrameworkCore;
+using NSubstitute;
+using ooapi.v5.core.Repositories;
+using ooapi.v5.core.UnitTests.Repositories.Helpers;
+using ooapi.v5.core.Utility;
+using ooapi.v5.Models;
+
+namespace ooapi.v5.core.UnitTests.Repositories;
+
+[TestFixture]
+public class OrganizationsRepositoryTests
+{
+    private readonly IFixture _fixture = new Fixture();
+
+    [Test]
+    public void GetAllOrderedBy_WithDataRequestParameters_ReturnsSet()
+    {
+        // Arrange
+        var organizations = new List<Organization>()
+        {
+            _fixture.Build<Organization>()
+                .Without(x => x.Addresses)
+                .Without(x => x.Address)
+                .Without(x => x.Parent)
+                .Without(x => x.Children)
```

Comment:
de withouts zijn nodig omdat AutoFixture die niet kan genereren? je zou in een TestSetup kunnen toevoegen:     fixture.Customize<Organization>(composer => composer.Without(p => p.Addresses)); Of als Organization in meer test classes wordt gebruikt een custom FixtureWithCustomizations class kunnen maken. https://stackoverflow.com/questions/38688932/how-to-use-autofixture-to-build-with-customized-properties-while-keeping-type-cu

Reply:
Hier ben ik mee bezig geweest maar het werkt nog niet met een basis FixtureWithCustomizations class bij mij. Dit mag door iemand anders opgepakt worden. De rename en andere PR comments zijn al gefixt.